build/publish/PublishNupkgToTransportFeed.targets

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <ItemsToPush Include="$(PackagesDirectory)/Microsoft.DotNet.Cli.Utils.*.nupkg" />
+    <ItemsToPush Include="$(PackagesDirectory)/Microsoft.DotNet.MSBuildSdkResolver.*.nupkg" />
```

Comment:
Can we unify the code where we determine which packages to push to storage and the transport feed?

Reply:
done...